src/position.cpp

```diff
@@ -510,8 +510,10 @@ Bitboard Position::slider_blockers(Bitboard sliders, Square s, Bitboard& pinners
     if (b && !more_than_one(b))
     {
         blockers |= b;
+        if (!pinners)
```

Comment:
Is `pinners` ever assigned a nonzero value?

Reply:
It is.